.github/workflows/python-test.yml

```diff
@@ -0,0 +1,96 @@
+name: CI Pipeline
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.8", "3.9", "3.10", "3.11", "3.12"]
```

Comment:
This also adds test cases for our full set of actively supported Python versions. Technically we support `>=3.6` but in a future release we will support only `>=3.8`.

Reply:
Definitely min 3.8 and max 3.12, but I’m not sure there’s much value for components to test everything between, since there’s very little Python code in the component

Reply:
I'll reduce the matrix to just 3.12 & 3.8